pkg/controller/platformfeatures_api.go

```diff
@@ -101,6 +102,55 @@ func (s *PlatformFeaturesApi) DeletePlatformFeatures(ctx context.Context, in *ed
 	return s.store.Delete(ctx, in, s.sync.SyncWait)
 }
 
+func (s *PlatformFeaturesApi) ShowPlatformFeaturesForZone(key *edgeproto.ZoneKey, cb edgeproto.PlatformFeaturesApi_ShowPlatformFeaturesForZoneServer) error {
```

Comment:
It looks like this returns a superset of features for all the cloudlets in this zone. For my understanding is the intent to return all the supported features and then use `potentialCloudlet` methods to get the cloudlet that support the features if one is required for an app, right?

Reply:
Actually the API is being used by the QA acceptance tests, to see if it should skip the test if the target cloudlet/zone doesn't support k8s or docker.